inz-demo/internal-affairs/Issues/name/description.xml

```diff
@@ -1,14 +1,14 @@
 <IssueSpecification version="4">
     <Name>
-        <en>Demo Fake Internal affairs full name</en>
-        <nl>Demo Fake Internal affairs full name</nl>
+        <en>Demo Internal Affairs full name</en>
+        <nl>Demo Internal Affairs full name</nl>
     </Name>
     <ShortName>
         <en>Full name</en>
         <nl>Name</nl>
     </ShortName>
     <SchemeManager>inz-demo</SchemeManager>
-    <IssuerID>fake-internalaffairs</IssuerID>
+    <IssuerID>internal-affairs</IssuerID>
```

Comment:
I think it's good to keep the internal affairs issuer in as later we might play with adding a new passport credential which also comes from them. Date of birth can also come from internal affairs as they have all the records of birth for NZ.